templates/events/components/toggle_active_flag_modal.html

```
@@ -1,5 +1,6 @@
 <dialog id="toggleEventModal-{{ event.id }}" class="modal">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)